test/grpc/BUILD.bazel

```
@@ -53,6 +53,36 @@ cc_binary(
     }),
 )
 
+cc_library(
+    name = "grpc_streaming",
```

Unless there is something that is keeping this from working it seems redundant and unnecessary to include the `grpc` prefix so let's drop.
```suggestion
    name = "streaming",
```